include/AutomatableModel.h

```diff
@@ -170,6 +170,18 @@ class LMMS_EXPORT AutomatableModel : public Model, public JournallingObject
 		return castValue<T>( m_value );
 	}
 
+	inline float getTrueValue(int frameOffset = 0) const
+	{
+		if (m_controllerConnection == nullptr && hasLinkedModels() == false)
+		{
+			return m_value;
+		}
+		else
```

Comment:
You don't really need an else here. The return takes care of it. That's the whole point

Reply:
Sorry, did it out of habit.

Reply:
You can do another simplification 
`if (!m_controllerConnection && !hasLinkedModels())`

Reply:
You can condense the entire thing to a single line and use ternary operator instead. This thought Crossed just now

Reply:
I know, I prefer to write it out, just for readability. I will change this.